source/application/AMPIC887DriverServer/AMGCS2.h

```diff
@@ -0,0 +1,107 @@
+#ifndef AMGCS2_H
+#define AMGCS2_H
+
+#include <QChar>
+/*!
+  * Namespace which contains all the enumerators for use with the GCS2.0 syntax
+  * provided by the PI C887.11 controllers.
+  */
+namespace AMGCS2 {
+
+	/***************
+	  * Enums
+	  ****************************/
+
+	/*!
+	  * Enumerates all the hexapod axes controlled by a C887.11 controller.
+	  */
+	enum Axis {
+		UnknownAxis = -1,
+		XAxis = 0,
```

Comment:
Thanks chaps. 
I think what I might do is make a note of this, and get to it at a later date. It will certainly improve efficiency, but feel it can be something done down the road.

If nobody minds I'll create an issue for it, but not include it in the Hexapod milestone (as it isn't fundamental to its workings). This way I won't forget to get to it later.